dll/shellext/fontext/fontext.rc

```diff
@@ -14,6 +14,9 @@
 #ifdef LANGUAGE_EN_US
     #include "lang/en-US.rc"
 #endif
+#ifdef LANGUAGE_RO_RO
+    #include "lang/ro-RO.rc"
+#endif
```

Comment:
Why is that, whereas there is just no romanian resource language for fontext?

Reply:
This file is exist:
https://github.com/reactos/reactos/blob/master/dll/shellext/fontext/lang/ro-RO.rc